test/services/markdown_service_test.rb

```diff
@@ -9,6 +9,14 @@ class MarkdownServiceTest < ActiveSupport::TestCase
     assert_equal(output, expected)
   end
 
+  test 'it parses emoji' do
```

Comment:
Is this a false positive? Happy to fix, but the other tests in this file make the same 'mistake'